x-pack/plugins/search_playground/public/plugin.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CoreSetup,
+  Plugin,
+  CoreStart,
+  AppMountParameters,
+  DEFAULT_APP_CATEGORIES,
+} from '@kbn/core/public';
+import { PLUGIN_NAME } from '../common';
+import { PlaygroundToolbar, PlaygroundProvider, Playground } from './embedable';
+import {
+  AppPluginStartDependencies,
+  SearchPlaygroundPluginSetup,
+  SearchPlaygroundPluginStart,
+} from './types';
+
+export class SearchPlaygroundPlugin
+  implements Plugin<SearchPlaygroundPluginSetup, SearchPlaygroundPluginStart>
+{
+  public setup(core: CoreSetup): SearchPlaygroundPluginSetup {
+    core.application.register({
+      id: 'playground',
+      appRoute: '/app/enterprise_search/playground',
```

Comment:
This should not contain `enterprise_search`
```suggestion
      appRoute: '/app/search_playground',
```

Reply:
Yeah, missed this since I tried to add this plugin from setup to ent_search